src/Sparrow.Server/Utils/DiskHelper.cs

```diff
@@ -10,9 +10,9 @@
 
 namespace Sparrow.Server.Utils
 {
-    public static class DiskSpaceChecker
+    public static class DiskHelper
```

Comment:
Can we call it `DiskUtils`?

Reply:
Not done?

Reply:
https://github.com/ravendb/ravendb/pull/14129#discussion_r881713228